README.md

```diff
@@ -42,6 +42,19 @@ This indicates that your plugin is calling `Project#allprojects()`, which violat
 and also that it calls `Task#getProject()`, which violates best practices when called from the context of a method
 annotated with `@TaskAction`.
 
+## Baselines
+
+In case there are many best practice violations in a project, it's worth generating a baseline to temporarily accept issues, and to prevent new ones from getting onto the main branch.
+
+To generate a baseline run the `bestPracticesBaseline` task:
+```shell
+./gradlew :plugin:bestPracticesBaseline
+```
+
+This will generate a file called: `best-practices-baseline.json` in the project directory.
+Version control this file, so gets propagated to CI and every developer.
```

Comment:
This file is not VCS friendly, because it's a single-line JSON. Please consider auto-formatting the file, and ensuring the the contents are also sorted by something stable.  
(e.g. `traces.sort { "${it.owner}/${it.name}/${it.descriptor}" }`)